client/packages/purchasing/src/purchase_order/ListView/OutstandingLines/OutstandingLines.tsx

```diff
@@ -44,115 +40,98 @@ const OutstandingLinesList = () => {
   const {
     query: { data, isError, isLoading },
   } = usePurchaseOrderLineList(listParams);
-  const pagination = { page, first, offset };
 
-  const columns = useColumns<PurchaseOrderLineFragment>(
-    [
+  const mrtColumns = useMemo(
+    (): ColumnDef<PurchaseOrderLineFragment>[] => [
```

Comment:
You should be able to enable sorting on these columns:

<img width="329" height="125" alt="Image" src="https://github.com/user-attachments/assets/58eb36d6-6436-4022-960d-c4053e70c050" />